_usage/locales.md

```diff
@@ -295,33 +302,30 @@ Variation-Selector Sensitivity (VSS) collations distinguish between
 ideographic variation selectors in Japanese collations
 `Japanese_Bushu_Kakusu_140` and `Japanese_XJIS_140`. A variation
 sequence is made up of a base character plus an additional variation
-selector. If you don\'t the select the `_VSS` option, the variation
-selector isn\'t considered in the comparison.
+selector. If you don't the select the `_vss` option, the variation
+selector isn't considered in the comparison.
 
 VSS collations will not be provided by Babelfish in the first
```

Comment:
and here also

Reply:
I agree that we should use 'current version' whenever possible...

Reply:
Done.